ui/components/app/token-list-display/token-list-display.scss

```
@@ -0,0 +1,43 @@
+.loading-span {
+  display: flex;
+  height: 250px;
+  align-items: center;
+  justify-content: center;
+  padding: 30px;
+}
+
+.token-list-item {
+  padding: 8px;
+  display: flex; 
+  flex-flow: row nowrap;
+  align-items: center;
+  cursor: pointer;
+  padding: 8px;
+
+  &:hover {
+    background-color: rgba($alto, 0.2);
+  }
+
+  &__data {
+    margin-left: 8px;
+  }
+
+  &__symbol {
+    @include Paragraph;
+    line-height: 140%;
+    margin-bottom: 2px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
+  &__balance {
+    @include H7;
+    display: flex;
+    flex-flow: row nowrap;
+  }
+  
+  &__balance__label {
+    margin-right: 0.25rem;
```

Any reason why you are using rems here? We are following the convention of using px values for margin and padding.

```suggestion
    margin-right: 4px;
```